crates/primitives/src/bytecode/eof/header.rs

```diff
@@ -242,7 +242,7 @@ mod tests {
         let (header, _) = EofHeader::decode(&input).unwrap();
         assert_eq!(header.types_size, 4);
         assert_eq!(header.code_sizes, vec![1]);
-        assert_eq!(header.container_sizes, vec![]);
+        assert_eq!(header.container_sizes, Vec::<u16>::new());
```

Comment:
I'm unsure why, but these assertions were causing compiler errors locally due to underspecified Vec types